src/trigger.js

```diff
@@ -0,0 +1,189 @@
+try {
```

Comment:
Write test code for `src/trigger.js`

e.g) https://github.com/sue445/chrome-gitlab-notifier/blob/2.2.0/test/popup_test.js

Reply:
added a test file, but it doesn't cover all new functions